### PR TITLE
React week2 homework + Fibonacci_B

### PR DESCRIPTION
**Pull request by CharlesKimMaina**

### CODE REVIEW

File: react/week2/fibonacci/src/components/FibonacciIncrement.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+export function FibonacciIncrement(prop) {
+  const increment = () => {
+    prop.setFiboList((previous) => {
+      const fibSeries = [...previous];     
+      for (let i = previous.length; i < previous.length + 10; i++) {
```
Comment (piyushpalod) [inline react/week2/fibonacci/src/components/FibonacciIncrement.js:8]: can you try without loop, as what Niels shows in class.

File: react/week2/todolist/src/components/Todos.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+   return (
+    <div className="App">      
+      <button onClick={addTodoItem}>Add to do</button>
+      <ul>
+        {todoList.map((todo) => {
+          return <div> 
+          <li key={todo.id}>{":"}{todo.description}</li> 
+          <button>done</button>     
+            <button>delete </button>
```
Comment (piyushpalod) [inline react/week2/todolist/src/components/Todos.js:46]: where is logic to delete todo list ?

**Review by piyushpalod — COMMENTED** [review]
Can you please check PR there was not logic or component for delete or update todo list.

File: react/week2/todolist/src/components/Todos.js (hunk 2 above)
Comment (CharlesKimMaina) [reply react/week2/todolist/src/components/Todos.js:46]: OKey,.thanks,.i will add it

Comment (CharlesKimMaina) [reply react/week2/todolist/src/components/Todos.js]: function deleteTodo(e) {
    const { id } = e.target.parentElement
    todoList.splice(id, 1)
    settodoList([...todoList]);
      }

Comment (CharlesKimMaina) [reply react/week2/todolist/src/components/Todos.js]: I have added it on the "reviews implented new PR",.Thanks

**Pull request merged**